pkg/helm/releases.go

```diff
@@ -37,3 +43,70 @@ func MakeHelmRelease(name, version, cluster, namespace string, helmRepository ty
 		},
 	}
 }
+
+// FindReleaseInNamespace iterates through a slice of HelmReleases to find one with a given name in a given namespace, and returns it with its index.
+func FindReleaseInNamespace(existingReleases []helmv2beta1.HelmRelease, name, ns string) (*helmv2beta1.HelmRelease, int, error) {
```

Comment:
```suggestion
func FindReleaseFromList(existingReleases []helmv2beta1.HelmRelease, name, ns string) (*helmv2beta1.HelmRelease, int, error) {
```

or something like that, the `Namespace` part seems a bit off

Reply:
completely changed this bit! pls review again